google/cloud/internal/retry_loop.h

```diff
@@ -101,12 +125,13 @@ auto RetryLoop(std::unique_ptr<RetryPolicy> retry_policy,
                Request const& request, char const* location)
     -> google::cloud::internal::invoke_result_t<Functor, grpc::ClientContext&,
                                                 Request const&> {
-  std::function<void(std::chrono::milliseconds)> sleeper =
-      [](std::chrono::milliseconds p) { std::this_thread::sleep_for(p); };
-  sleeper = MakeTracedSleeper(CurrentOptions(), std::move(sleeper));
-  return RetryLoopImpl(*retry_policy, *backoff_policy, idempotency,
-                       std::forward<Functor>(functor), request, location,
-                       std::move(sleeper));
+  auto wrapper = [&functor](grpc::ClientContext& context, Options const&,
```

Comment:
I think this thing was called `with_options` in the `RestRetryLoop`..... but the plan is to delete this overload, so who cares.